eng/devices/windows.cake

```diff
@@ -332,18 +335,23 @@ Task("uitest")
 	var name = System.IO.Path.GetFileNameWithoutExtension(PROJECT.FullPath);
 	var binlog = $"{BINLOG_DIR}/{name}-{CONFIGURATION}-windows.binlog";
 
-	var dd = MakeAbsolute(Directory("../../bin/dotnet/"));
-	Information("DOTNET_PATH: {0}", dd);
+	Information("old dotnet root: {0}", DOTNET_ROOT);
+	Information("old dotnet path: {0}", DOTNET_PATH);
 
-	var toolPath = $"{dd}/dotnet.exe";
+	var localDotnetRoot = MakeAbsolute(Directory("../../bin/dotnet/"));
+	Information("new dotnet root: {0}", localDotnetRoot);
 
-	Information("toolPath: {0}", toolPath);
+	DOTNET_ROOT = localDotnetRoot.ToString();
```

Comment:
Since the env var `$DOTNET_ROOT` doesn't even work anymore in .NET 8, should we consider just naming this variable something else to avoid confusion?